src/lib/wallet/GoodWalletClass.js

```diff
@@ -626,6 +643,75 @@ export class GoodWallet {
     }
   }
 
+  async getActiveClaimers(): Promise<number> {
+    try {
+      const activeUsersCount = await this.UBIContract.methods.activeUsersCount().call()
+      return activeUsersCount.toNumber()
+    } catch (exception) {
+      const { message } = exception
+
+      log.warn('getActiveClaimers failed', message, exception)
+      throw exception
+    }
+  }
+
+  async getAvailableDistribution(): Promise<number> {
+    try {
+      let currentDay = await this.UBIContract.methods.currentDay().call()
+      currentDay = currentDay.toNumber()
+      const dailyUBIHistory = await this.UBIContract.methods.dailyUBIHistory(currentDay).call()
+      return dailyUBIHistory.openAmount.toNumber()
+    } catch (exception) {
+      const { message } = exception
+      log.warn('getTodayDistribution failed', message, exception)
+      throw exception
+    }
+  }
+
+  async getTotalFundsStaked(): Promise<number> {
+    try {
+      let totalFundsStaked = await this.SimpleDaiStaking.methods.totalStaked().call()
+      return this.web3Mainnet.utils.fromWei(totalFundsStaked)
+    } catch (exception) {
+      const { message } = exception
+      log.warn('getTotalFundsStaked failed', message, exception)
+      throw exception
+    }
+  }
+
+  async getInterestCollected(): Promise<number> {
+    try {
+      const toBlock = await this.web3Mainnet.eth.getBlockNumber()
+      const fromBlock = parseInt(toBlock) - parseInt(Config.interestCollectedInterval)
+      const InterestCollectedEventsFilter = {
+        fromBlock,
+        toBlock,
+      }
+
+      const events = await this.SimpleDaiStaking.getPastEvents(
+        'InterestCollected',
+        InterestCollectedEventsFilter,
+      ).catch(e => {
+        //just warn about block not  found which is recoverable
+        const logFunc = e.code === -32000 ? log.warn : log.error
+        logFunc('InterestCollectedEvents failed:', e.message, e, {
+          category: ExceptionCategory.Blockhain,
+        })
+        return []
+      })
+      let interest = 0
+      if (events.length > 0) {
```

Comment:
no need for if condition.
fix logic to get interest
interest  =  fromWei(get(last....